app/utils/print/sections/AnswerRowConverter.scala

```diff
@@ -193,6 +193,21 @@ class AnswerRowConverter @Inject()(checkAnswersFormatters: CheckAnswersFormatter
     }
   }
 
+  def numberOfBeneficiariesQuestion(query: Gettable[HowManyBeneficiaries],
+                        userAnswers: UserAnswers,
+                        labelKey: String,
+                        messageArg: String = "")
+                       (implicit messages: Messages, rds: Reads[HowManyBeneficiaries]): Option[AnswerRow] = {
```

Comment:
One final thing - again very picky! You don't need to provide an implicit reads if the type is known